package.json

```diff
@@ -22,7 +22,7 @@
     "url": "https://github.com/Level/packager.git"
   },
   "scripts": {
-    "test": "standard && node test.js && rm -rf level-test*"
+    "test": "standard && node test.js && rm -rf abstract/level-test*"
```

Comment:
Would it be possible to move `rm -rf` to code instead? With `rimraf`.

Or don't use the cwd, but a temp folder.

Reply:
No need tweaking this these tests since they will be rewritten anyway. https://github.com/Level/packager/issues/62